org.mwc.debrief.data_feed/src/org/mwc/debrief/data_feed/views/DataFeed.java

```diff
@@ -340,8 +362,8 @@ public void setFocus()
 	public void insertData(final String data)

```

Comment:
psst @ayeshare, why has this file been changed?

Reply:
because I removed the throws IOException, because exception was never thrown